wazo_acceptance/steps/phone_call.py

```diff
@@ -64,6 +65,7 @@ def step_a_calls_exten_and_waits_for_x_seconds(context, tracking_id, exten, time
 @when('I wait "{seconds}" seconds for the call to be forwarded')
 @when('I wait "{seconds}" seconds for the end of ringing time')
 @when('I wait "{seconds}" seconds for the timeout to not expire')
+@when('I wait "{seconds}" seconds for wazo-calld\'s load disappears')
```

Comment:
I think it would be best to call it something like `'I wait "{seconds}" seconds for wazo-calld's load to drop`, because it does not really disappear :smile: